src/aap_eda/api/views/activation.py

```diff
@@ -80,6 +82,31 @@ def create(self, request):
 
         response = serializer.create(serializer.validated_data)
 
+        if response.webhooks:
+            extra_var = {}
+            if response.extra_var_id:
+                extra_var_obj = models.ExtraVar.objects.get(
+                    pk=response.extra_var_id
+                )
+                extra_var = yaml.safe_load(extra_var_obj.extra_var)
+
+            extra_var["EDA_WEBHOOK_CHANNELS"] = [
+                webhook.channel_name for webhook in response.webhooks.all()
+            ]
+            extra_var["EDA_PG_NOTIFY_DSN"] = settings.PG_NOTIFY_DSN_CLIENT
+            if response.extra_var_id:
+                extra_var_obj = models.ExtraVar.objects.get(
+                    pk=response.extra_var_id
+                )
```

Comment:
This `get()` of the extra var looks to be redundant based on lines 87-91 which has already retrieved it when `response.extra_var_id` is set.